src/player/tests.rs

```diff
@@ -0,0 +1,52 @@
+use crate::player::{PLAYER_SPEED, Player};
+
+#[test]
+fn stop_moving_stops_the_player() {
```

Comment:
`when_player_is_moving_stops_it`?

Reply:
Yeah, it was bad hahah, I've fixed it with the name you've proposed :wink: